modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -127,122 +156,118 @@ endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
+<1> The Cluster API Operator CR is #required#.
+<2> The Machine API Operator CR is required.
+<3> The Cloud Credential Operator CR is required.
+<4> The Image Registry Operator CR is required.
+<5> The Ingress Operator CR is required.
+<6> The Network Operator CR is required.
+<7> The AWS Elastic Block Store CSI Driver Operator CR is an optional component and might be disabled in your cluster.
 endif::aws-sts[]
 ifdef::google-cloud-platform[]
-. If your cluster uses cluster capabilities to disable one or more optional components, delete the `CredentialsRequest` custom resources for any disabled components.
-+
-.Example `credrequests` directory contents for {product-title} 4.12 on GCP
-+
+.Example `credrequests` directory contents for {product-title} 4.14 on GCP
 [source,terminal]
 ----
 0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml <1>
-0000_30_machine-api-operator_00_credentials-request.yaml <2>
-0000_50_cloud-credential-operator_05-gcp-ro-credentialsrequest.yaml <3>
-0000_50_cluster-image-registry-operator_01-registry-credentials-request-gcs.yaml <4>
+0000_30_cluster-api_00_credentials-request.yaml <2>
+0000_30_machine-api-operator_00_credentials-request.yaml <3>
+0000_50_cloud-credential-operator_05-gcp-ro-credentialsrequest.yaml <4>
+0000_50_cluster-image-registry-operator_01-registry-credentials-request-gcs.yaml <5>
+0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml <6>
+0000_50_cluster-network-operator_02-cncc-credentials.yaml <7>
+0000_50_cluster-storage-operator_03_credentials_request_gcp.yaml <8>
+----
+<1> The Cloud Controller Manager Operator CR is required.
+<2> The Cluster API Operator CR is #required#.
+<3> The Machine API Operator CR is required.
+<4> The Cloud Credential Operator CR is required.
+<5> The Image Registry Operator CR is required.
+<6> The Ingress Operator CR is required.
+<7> The Network Operator CR is required.
+<8> The GCP PD CSI Driver Operator CR is an optional component and might be disabled in your cluster.
+endif::google-cloud-platform[]
+ifdef::azure-workload-id[]
+.Example `credrequests` directory contents for {product-title} 4.14 on Azure
+[source,terminal]
+----
+0000_26_cloud-controller-manager-operator_14_credentialsrequest-azure.yaml <1>
+0000_30_cluster-api_00_credentials-request.yaml <2>
+0000_30_machine-api-operator_00_credentials-request.yaml <3>
+0000_50_cluster-image-registry-operator_01-registry-credentials-request-azure.yaml <4>
 0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml <5>
 0000_50_cluster-network-operator_02-cncc-credentials.yaml <6>
-0000_50_cluster-storage-operator_03_credentials_request_gcp.yaml <7>
+0000_50_cluster-storage-operator_03_credentials_request_azure_file.yaml <7>
+0000_50_cluster-storage-operator_03_credentials_request_azure.yaml <8>
 ----
-+
 <1> The Cloud Controller Manager Operator CR is required.
+<2> The Cluster API Operator CR is #required#.
```

Comment:
Per Andrew during standup today, CAPI won't work with AD Workload ID and should be documented accordingly here.

Reply:
This should be irrelevant due to the new `--included` flag. Leaving this PR alone because that work is in #62148